### PR TITLE
9519 edit recommendations

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/9519

This PR makes Fatality Board Review Recommendations:
- editable
- only viewable by those with admin or IT Supervisor roles
- only apply to crashes with `atd_fatality_count > 0`

## Testing
**URL to test:** 
https://deploy-preview-1091--atd-vze-staging.netlify.app/#/crashes/

**Steps to test:**
1. Go to a crash details page that doesn't already have recommendation details
2. To see the recommendations card, you must:
  - Be logged in with an account that is an admin or IT Supervisor
  - Be looking at a crash with `atd_fatality_count` > 0 (you can update this field to switch the card on and off)
3. To create the initial record, you can:
  - Select a **Coordination Partner** from the dropdown
  - Select a **Status** from the dropdown
  - Click "Add" next to the **Recommendation** field and click the checkbox to save
  - Click "Add" next to the **Updates** field and click the checkbox to save
4. After one of these four actions, the recommendation record exists and further updates will be edits to the existing record
5. Test updating the values in any of the fields and you will see the value persist
6. Test the cancel button when editing either **Recommendation** or **Updates** to make sure that the value is not edited even if you update it in the text area input

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/Notes.js
Comment (mddilley) [inline atd-vze/src/views/Crashes/Notes.js:124]: This all just to quiet some console warnings about style names

File: atd-vze/src/views/Crashes/Recommendations.js
Comment (mddilley) [inline atd-vze/src/views/Crashes/Recommendations.js:1]: This file was moved to a new folder before I touched this code for organization which I totally support ❤️

File: atd-vze/src/views/Locations/LocationNotes.js
Comment (mddilley) [inline atd-vze/src/views/Locations/LocationNotes.js:85]: I'll test out Location notes, but I suspect that Rose was making a small fix here. 👍

**Review by frankhereford — APPROVED** [review]
collab

This works so well, and I know the VZ team will get a huge boon from being able to record outcomes and agreements in these fields. Thank you both for this work!

Test & code reviewed 🚀🚀

**Review by chiaberry — APPROVED** [review]

**Comment by mddilley** [comment]
@patrickm02L I made the dropdown improvements that we talked about earlier today:
- Made the text in the dropdown button align left just like the text dropdown choices
- Made the dropdown menu choices the same width as the dropdown button

I'm going to take one more look at the wrap that happens with the **Coordination Partner** label and dropdown between small and large breakpoints. Thanks again for the feedback!

**Comment by patrickm02L** [comment]
Overall this works beautifully. 

I have a **_Change Request_** - Ability to select no option from drop downs. I'll wait for this update before approval.

There's a **_Quirk_** where the crash page loads to the note/recommendation section rather than the top of the page (not sure if this is caused by this feature). Do you have any insights into why this is happening?

I've also added a few notes about future Issues (that's to keep me accountable, you do not have to address them).

## Video Break Down

**Gos**
- Recommendations section only shows up when there are fatalities (:12)
- The drop downs look nice! (1:38)

**Change**
- Ability to select no option from drop down menu, to mimic the default blank setting. Otherwise, if you start to fill out recommendations and select a drop down you're stuck with whatever you choose. (2:00)

**Quirk**
- Some of the crash records jump straight to the notes/recommendations section. When the crash page is initially opened it should default to the top of the page. (:54)

**Future Issues**
- Formatting of text in Updates text box (2:41)
- Button spacing (4:27)


https://user-images.githubusercontent.com/5566727/186908350-8318f83e-00c9-4ef5-9e0f-8da98425d8ff.mp4

**Comment by mddilley** [comment]
@patrickm02L Just saw this feedback, I'm on it!

**Review by patrickm02L — APPROVED** [review]
Everything checks out! Thanks Mike for 🪨 this! You 🤘!

File: atd-vze/src/views/Crashes/Crash.js
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crash.js:71]: 👍, you could also double bang here ‼️

File: atd-vze/src/views/Crashes/Crash.js
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crash.js:73]: 👌✨

**Review by mateoclarke — APPROVED** [review]
This looks great! Going to go ahead and merge this and coordinate with Patrick when to launch 🚀

**Pull request merged**
